### PR TITLE
QR Code, verification code views

### PR DESCRIPTION
**Pull request by Velin92**
The PINTextField has been reused for the verification code

### CODE REVIEW

File: ElementX/Sources/Mocks/Generated/GeneratedMocks.swift [hunk 1]
```diff
@@ -903,7 +903,30 @@ class ApplicationMock: ApplicationProtocol {
...
+            if Thread.isMainThread {
+                openAppSettingsUnderlyingCallsCount = newValue
+            } else {
+                DispatchQueue.main.sync {
+                    openAppSettingsUnderlyingCallsCount = newValue
+                }
+            }
+        }
+    }
     var openAppSettingsCalled: Bool {
```
Comment (Velin92) [inline ElementX/Sources/Mocks/Generated/GeneratedMocks.swift:930]: Not sure why this got generated, I imagine the stencil has been recently changed

Comment (stefanceriu) [reply ElementX/Sources/Mocks/Generated/GeneratedMocks.swift:930]: Yeah, all good, I recently changed this to make the generated mocks *more* thread safe (probably incomplete)

**Review by pixlwave — APPROVED** [review]
Very nice 👌

File: ElementX/Sources/Screens/AppLock/AppLockSetupPINScreen/View/PINTextField.swift [hunk 2]
```diff
@@ -16,15 +16,22 @@
 
 import SwiftUI
 
+enum PINDigitFieldSize {
+    case small
+    case medium
+}
```
[the remark this answers is not in the document]
Comment (Velin92) [reply ElementX/Sources/Screens/AppLock/AppLockSetupPINScreen/View/PINTextField.swift:22]: this means I need to make the struct not private is that okay?

[the remark this answers is not in the document]
Comment (pixlwave) [reply ElementX/Sources/Screens/AppLock/AppLockSetupPINScreen/View/PINTextField.swift:22]: Ah now I understand. I'm ok with it either way tbh, whatever your prefer 👍

**Pull request merged**